packages/neuron-ui/src/components/NetworkEditorDialog/index.tsx

```diff
@@ -65,11 +68,14 @@ const NetworkEditorDialog = ({
         dataset: { field = '' },
       } = e.target as HTMLInputElement
       let error = ''
+      let fieldValue = value as string | number
```

Comment:
Type of `value` is `string`, type of `fieldValue` is `string | number`, so it should be

```ts
let fieldValue: string | number = value
```